appdata-patch.patch

```diff
@@ -0,0 +1,46 @@
+From b9f48bf5036c54bf9dd4e6ad38634c9ed6c9ee85 Mon Sep 17 00:00:00 2001
+From: bbhtt <62639087+bbhtt@users.noreply.github.com>
+Date: Sun, 31 Jul 2022 12:03:13 +0530
+Subject: [PATCH] appdata patch
+
+---
+ data/mx.pwmc.Svgvi.appdata.xml.in | 14 +++++++-------
+ 1 file changed, 7 insertions(+), 7 deletions(-)
+
+diff --git a/data/mx.pwmc.Svgvi.appdata.xml.in b/data/mx.pwmc.Svgvi.appdata.xml.in
+index ff451d1..003566c 100644
+--- a/data/mx.pwmc.Svgvi.appdata.xml.in
++++ b/data/mx.pwmc.Svgvi.appdata.xml.in
+@@ -1,22 +1,22 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+   <!-- Copyright 2019 PWMC Services S.A.S. de C.V. -->
+-	<id>mx.pwmc.Svgvi.desktop</id>
+-	<metadata_license>CC0-1.0</metadata_license>
+-	<project_license>GPL-3.0-or-later</project_license>
++  <id>mx.pwmc.Svgvi.desktop</id>
```

Comment:
```suggestion
+  <id>mx.pwmc.Svgvi</id>
```

Don't put `desktop` in the app-id.

It should be removed upstream too.

Reply:
It was a whitespace change there I think. I'll update the patch to remove the .desktop suffix.

Reply:
Ok I updated the appdata with appstream-util upgrade and removed the desktop suffix.